addons/account/wizard/account_payment_register.py

```diff
@@ -116,9 +116,54 @@ class AccountPaymentRegister(models.TransientModel):
         help="Technical field used to know whether the field `partner_bank_id` needs to be required or not in the payments form views")
     country_code = fields.Char(related='company_id.account_fiscal_country_id.code', readonly=True)
 
+    # == Early Payment Discount ==
+    has_epd_and_is_in_time = fields.Boolean(
+        compute='_compute_has_epd_and_is_in_time',
+        readonly=True,
+        store=False,
+        help="Indicates whether or not an early payment discount is available and if the conditions required to activate it are fulfilled",
+    )
+
+    show_epd_button = fields.Boolean(compute="_compute_show_epd_button")
```

Comment:
epd is shorter but it should something more explicit like show_early_pay_discount_button?